### ISSUE TITLE
Looking for collaborators

### ISSUE THREAD

**Issue by huafu** [huafu/ember-google-map#102]
As you could all see, I have no time to work on this anymore...

If some of you still use it and anyone is interested in becoming a collaborator, please lemme know!

/cc @dioptre @lukemcfarlane @maxbritto ...

**Comment by maxbritto** [comment]
Hello @huafu , sorry I'm no longer working on an emberjs project. Hope you'll find someone to take care of this.

**Comment by huafu** [comment]
Haha, same here, I moved over to some other framework. Tho I'd like to see what became Ember nowadays, I have NO TIME for it :-/

**Comment by maxbritto** [comment]
Yes I miss ember! I wish it would have gained in popularity and I could have stayed on it. I have to come back to web development for a new projet and I don't really like those new frameworks. Which one did you choose ?

**Comment by huafu** [comment]
I chose Vue and after playing a lot with it I really like Vue+TypeScript+JSX (well, TSX). But Vue SFC (single file components) in `.vue` files are nice as well.

If you're looking for a freaking amazing framework using view and having a full set of tools such as builder for web, mobile, ..., huge amount of nice components, ... https://quasar-framework.org/

I like Vue because it's very light, fast and also was easy to learn (at least from my point of view)

I love Ember for its ... hmm can't remember the English term for this... you have to do it this way and not another, organize your files this way, ... I don't know how it is now, but I remember it was, like Angular, way too heavy. VueJS is less than 60k minified while others are way more than the double

**Comment by maxbritto** [comment]
Thanks for the feedback! I was also going towards Vue, so this is reassuring to see you went down this path and ended up happy with it. Didn't know Quasar but it looks awesome! Thanks!

I think the word you're looking for regarding Ember is "opinionated" or "convention over configuration". Loved that too, once I got to understand it fully.

**Comment by huafu** [comment]
Yeah, it was on the tip of my tongue. The exact one I was looking for is "convention over configuration", but "opinionated"  actually describes better what I meant ;-)
